src/getList/index.ts

```diff
@@ -88,4 +104,12 @@ const getFilter = async (
   return result
 }
 
-export type FiltersOption = Record<string, (value: any) => any>
+
+const populateAdditionalFields =
```

Comment:
Do you have an example of use case here ? 
Also, I find it a bit counter intuitive to name this `populateAdditionalFields` as you are not really populating anything. Maybe the wording could be update for something like `computeAdditionalFields` ?